aesara/graph/features.py

```diff
@@ -559,7 +559,7 @@ def replace_all_validate(
 
         for r, new_r in replacements:
             try:
-                fgraph.replace(r, new_r, reason=reason, verbose=False, **kwargs)
+                fgraph.replace(r, new_r, reason=reason, verbose=verbose, **kwargs)
```

Comment:
```suggestion
                fgraph.replace(r, new_r, reason=reason, verbose=False, **kwargs)
```

`verbose` is set to `False` here because this method handles that functionality itself.